src/harness/client.ts

```diff
@@ -1009,6 +1012,22 @@ export class SessionClient implements LanguageService {
         return getSupportedCodeFixes();
     }
 
+    getPostPasteImportFixes(targetFile: string, pastes: Array<{text: string; range: TextRange}>, _preferences: UserPreferences, _formatOptions: FormatCodeSettings, originalFile?: string, copyRange?: CopyRange): PostPasteImportFixes{    
+        const args = this.createFileLocationOrRangeRequestArgs(pastes[0].range, targetFile) as protocol.GetPostPasteImportFixesRequestArgs;

```

Comment:
I'm not sure what the range should be passed in here to `createFileLocationOrRangeRequestArgs` . Is this going to be the position that the paste is called at and passes in as a separate parameter in the api?